timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/model/GraphReadUtils.java

```diff
@@ -13,8 +13,10 @@
 //Most database access should probably be done through the tinkerpopjsoncrudservice
 public class GraphReadUtils {
 
+  public static final String TYPES_PROP = "types";
+
   public static Optional<Try<String[]>> getEntityTypes(Element element) {
-    return getProp(element, "types", String.class)
+    return getProp(element, TYPES_PROP, String.class)
```

Comment:
de "types" string komt 59 keer voor in de codebase. ik denk niet dat deze constante veel nut heeft zo. Voor tim_id doen we het weer niet.

Ik snap wel enigszins de waarde van dit soort constanten, maar zullen we ze dan allemaal vervangen? (mag na het mergen van deze PR) en zullen we ze dan als naam de inhoud van de string geven? Dus `TYPES`, `TIM_ID` `REV`. Als programmeur moet ik ook vaak in de database naar de onderliggende representatie kijken dus als we ze anders noemen moet ik twee woorden voor hetzelfde onthouden.

Is er een java-waardige manier om ze naar lowercase te krijgen?